### DISCUSSION TITLE
Why does the image replication alternate between being successful and encountering a 401 error?

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#20004] [Q&A]
Hi,
Sometimes image replication succeeds, and sometimes it fails with 401 error. What is the reason?
```
failed to pull the manifest of artifact ttt2222/redis:latest_0214_2: http status code: 401, body: {"errors":[{"code":"UNAUTHORIZED","message":"unauthorized to access repository: ttt2222/redis, action: pull: unauthorized to access repository: ttt2222/redis, action: pull"}]}
```

**Comment by zyyw** [comment]
@hyeyoung-leee would you mind sharing with us the Harbor version you are using when you get the above error? Thanks

**Reply by zyyw** [reply]
Could you please also share with us how you deployed your Harbor instance?

**Comment by hyeyoung-leee** [comment]
> @hyeyoung-leee would you mind sharing with us the Harbor version you are using when you get the above error? Thanks

@zyyw 
I apologize for the confusion.
The issue stemmed from the Harbor configuration, specifically from configuring two Harbors to point to the same PostgreSQL database.
I kept two Harbors for the migration process, but removing one of them solved the problem.
Thanks.
